app/models/event.rb

```diff
@@ -888,6 +892,10 @@ def point_of_contact_history
                                   .filter_map { |(old_id, _new_id)| User.find_by(id: old_id) }
   end
 
+  def linked_discord_guild?
+    discord_guild_id.present?
+  end
```

Comment:
We rename this and it's usages to `has_discord_guild?`?
```suggestion
  def has_discord_guild?
    discord_guild_id.present?
  end
```

Reply:
[`cf97b36` (#11791)](https://github.com/hackclub/hcb/pull/11791/commits/cf97b36c41887cc6ba22284da3bbfd642bbe177d)